@ctb One test from `tests/test_cmd_signature.py` is failing. This is the error message:
```
>           raise SourmashCommandFailed(self.last_result.err)

E           == This is sourmash version 4.1.2.dev6+g87a908c.d20210609. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==
E           
E           Traceback (most recent call last):
E             File "/Users/keya/sourmash/tests/sourmash_tst_utils.py", line 105, in runscript
E               status = _runscript(scriptname)
E             File "/Users/keya/sourmash/tests/sourmash_tst_utils.py", line 47, in _runscript
E               pkg_resources.load_entry_point("sourmash", 'console_scripts', scriptname)()
E             File "/Users/keya/sourmash/src/sourmash/__main__.py", line 13, in main
E               return mainmethod(args)
E             File "/Users/keya/sourmash/src/sourmash/cli/sketch/translate.py", line 97, in main
E               return sourmash.command_sketch.translate(args)
E             File "/Users/keya/sourmash/src/sourmash/command_sketch.py", line 259, in translate
E               raise ValueError("cannot set both --dayhoff and --hp")
E           ValueError: cannot set both --dayhoff and --hp

tests/sourmash_tst_utils.py:195: SourmashCommandFailed
```
I can't seem to figure out how to fix this.

--- a/tests/test_cmd_signature.py
+++ b/tests/test_cmd_signature.py
@@ -2584,11 +2584,10 @@ def test_sig_describe_dayhoff(c):
 def test_sig_describe_1_hp(c):
     # get basic info on a signature
     testdata = utils.get_test_data('short.fa')
-    # c.run_sourmash('compute', '-k', '21,30',
-    #                '--dayhoff', '--hp', '--protein',
-    #                '--dna',
-    #                testdata)
-    c.run_sourmash('sketch', 'translate', '-p', 'k=21,num=500', '-p', 'k=30,num=500', '--dayhoff', '--hp', testdata)             
+    c.run_sourmash('compute', '-k', '21,30',
+                   '--dayhoff', '--hp', '--protein',
+                   '--dna',
+                   testdata) 
     # stdout should be new signature
     computed_sig = os.path.join(c.location, 'short.fa.sig')
     c.run_sourmash('sig', 'describe', computed_sig)

--- a/tests/test_cmd_signature.py
+++ b/tests/test_cmd_signature.py
@@ -2587,7 +2587,7 @@ def test_sig_describe_1_hp(c):
     c.run_sourmash('compute', '-k', '21,30',
                    '--dayhoff', '--hp', '--protein',
                    '--dna',
-                   testdata) 
+                   testdata)
     # stdout should be new signature
     computed_sig = os.path.join(c.location, 'short.fa.sig')
     c.run_sourmash('sig', 'describe', computed_sig)

--- a/tests/test_lca.py
+++ b/tests/test_lca.py
@@ -2101,7 +2101,7 @@ def test_compare_csv_real():
 def test_incompat_lca_db_ksize_2(c):
     # test on gather - create a database with ksize of 25
     testdata1 = utils.get_test_data('lca/TARA_ASE_MAG_00031.fa.gz')
-    c.run_sourmash('compute', '-k', '25', '--scaled', '1000', testdata1,
+    c.run_sourmash('sketch', 'dna', '-p', 'k=25,scaled=1000', testdata1,
                    '-o', 'test_db.sig')
     print(c)
 

